src/screens/AdminPortal/OrganizationPeople/addMember/AddMember.tsx

```
@@ -5,40 +5,40 @@
  * 1. Adding an existing user from the user list.
  * 2. Creating a new user and adding them to the organization.
  *
- * Features
+ * @Features
```

ESLint

[error] tsdoc-undefined-tag: The TSDoc tag "@Features" is not defined in this configuration

(tsdoc/syntax)